src/modules/fw_pos_control_l1/fw_pos_control_l1_main.cpp

```diff
@@ -1686,7 +1524,7 @@ FixedwingPositionControl::control_position(const math::Vector<2> &current_positi
 				 * if current position is below the slope continue at previous wp altitude
 				 * until the intersection with slope
 				 * */
-				float altitude_desired_rel;
+				float altitude_desired_rel{0.0f};
```

Comment:
It's c++11 uniform initialization. Admittedly it looks a bit odd at first.
I want to work towards requiring everything be initialized, and it's easier and more compact to do it in the class. There are some side benefits (consistency, type safety, clang-tidy support, preferred by the cppcoreguidelines, etc), but that's my main interest. 
Eventually we can start using clang tooling to enforce initialization.